src/daft-image/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+mod counting_writer;
+mod image_buffer;
+mod iters;
+pub mod ops;
+pub use counting_writer::CountingWriter;
+pub use image_buffer::DaftImageBuffer;
+pub mod series;
+
+#[cfg(feature = "python")]
+mod python;
+
+#[cfg(feature = "python")]
+pub use python::*;
```

Comment:
WDYT instead of doing `pub mod python` and then users can do `use daft_image::python::...` if they want to access the inner stuff?